interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -847,7 +847,44 @@ def test_nondimensional(self):
 
 class TestPengRobinsonPhase(utilities.CanteraTest):
     def setup(self):
-        self.gas = ct.Solution('co2_PR_exampl.yaml','CO2-PR')
+        self.gas = ct.Solution('co2_PR_example.yaml','CO2-PR')
+
+    def test_setSV(self):
+        """
+        Set state in terms of (s,v) 
+        """
+        self.gas = ct.Solution('co2_PR_example.yaml','CO2-PR')
```

Comment:
If you create `self.gas` in the setup function, you shouldn't have to recreate it in all the test functions as well.

Reply:
Corrected.